website_country_localized_pages/__init__.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2015 Agile Business Group (<http://www.agilebg.com>)
+#
+#    This program is free software: you can redistribute it and/or modify
```

Comment:
You can make this header lighter according last proposals (only copyright and AGPL-3 mention pointing to manifest)